pkgs/by-name/vk/vkd3d-proton/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, glslang
+, meson
+, ninja
+, stdenv
+, wine
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vkd3d-proton";
+  version = "2.11.1-unstable-2024-03-07";
+
+  src = fetchFromGitHub {
+    owner = "HansKristian-Work";
+    repo = "vkd3d-proton";
+    rev = "ffb04fb7091f932ca95da236d63ab9a4064d6443";
+    fetchSubmodules = true;
+    hash = "sha256-xkDWquEbsJD6b8vvZsMHiqN0IgyyNY0G36ol8Sq041U=";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    glslang
+    meson
+    ninja
+    wine
+  ];
+
+  strictDeps = true;
+
+  prePatch = let
```

```suggestion
  postPatch = let
```
We don't want to break patches